### ISSUE TITLE
Suggestion: rivers in world gen and, obviously, BOATS

### ISSUE THREAD

**Issue by kiwinuken** [voxeltycoon/suggestions#247]
In old TTD, boats were always underused. Most maps wouldn't feature an interesting terrain to make them useful enough. The only thing that made them necessary was oil rigs, but if you had access to oil fields in land, that was always preferable.

If world gen can be tweaked to make rivers of various widths appear connecting regions, or even as borders between them - which would make borders look more organic and meaningful -, that would make waterways a very interesting transport modal. IRL it is used to transport vast quantities of goods, specially commodities, in a much cheaper way than even trains. The sluggish speed most ferries feature is compensated by the ludicrous amounts of cargo they hold, and they serve as a kind of "moving warehouses": goods stay in transport long enough to make so that they arrive just in time to be used, cutting further costs with storage.

It would be interesting to see a more diverse landscape that features river systems and present construction challenges, but that would also make boats an interesting choice for transportation if it's implemented.

**Comment by RazzleDazzleDaeris** [comment]
I believe boats - in some capacity - are on the roadmap.

**Comment by victorleblais** [comment]
Both of those things are on the roadmap
![image](https://user-images.githubusercontent.com/4365075/115475953-0d05f400-a241-11eb-8e88-0515b1dc28ab.png)

![image](https://user-images.githubusercontent.com/4365075/115475960-11321180-a241-11eb-8b7e-466b36f4a1e1.png)

**Comment by kiwinuken** [comment]
I saw that boats were planned on the road map, I added it just to link both features more clearly, because when people think of boats on a game like VT they may not have rivers in mind. Landscape features totally escaped my attention, though. Sorry for that.

I guess this can be closed, unless you people want to let it open to hold discussions about those features in an integrated way (it might be worth to consider landscape features as the concept upon which different transport modals integrate, as solutions to problems that arise from distance, geographic features and infrastructure costs).
